ams/client.go

```diff
@@ -149,18 +150,26 @@ func (c *Client) GetSubscription(subscriptionId string) (*v1.Subscription, error
 	return resp.Body(), nil
 }
 
-func (c *Client) GetSubscriptions(organizationId string, size, page int) (*v1.SubscriptionList, error) {
+func (c *Client) GetSubscriptions(organizationId string, statuses []string, size, page int) (*v1.SubscriptionList, error) {
 	amsOrgId, err := c.ConvertUserOrgId(organizationId)
 	if err != nil {
 		return nil, err
 	}
-	q := "plan.id LIKE 'AnsibleWisdom'"
-	q += " AND "
-	q += fmt.Sprintf("organization_id = '%s'", amsOrgId)
+
+	queryBuilder := NewQueryBuilder().
+		Like("plan.id", "AnsibleWisdom").
+		And().
+		Equals("organization_id", amsOrgId)
```

Comment:
Since this is being interpolated into a string, ensure that the value is safe

Reply:
Also, I appreciate that it's been unvalidated in the past, sorry for that :)

Reply:
Maybe allowlist something like this `/[a-zA-Z0-9]+/`.

Reply:
Though, now that you remind me maybe it's not really helpful.

Reply:
sounds good :rocket: updated

Reply:
I also validated the user org id from the user request, since we also query ams with that